addons/project_todo/static/tests/todo_conversion_form_view.test.js

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from "@odoo/hoot";
+import { queryAll } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { mountView, contains, onRpc } from "@web/../tests/web_test_helpers";
+import { defineTodoModels } from "./todo_mock_models";
+import { click } from "@mail/../tests/mail_test_helpers";
```

Comment:
Same here, and we should use the click function on the web instead of mail.

import { click } from "@odoo/hoot-dom";